mlflow/recipes/steps/split.py

```diff
@@ -331,9 +397,12 @@ def _run(self, output_directory):
         self.num_dropped_rows = raw_input_num_rows - len(input_df)
 
         # split dataset
-        test_df, train_df, validation_df = _run_split(
-            self.task, input_df, self.split_ratios, self.target_col
-        )
+        if self.step_config["using"] == "custom":
+            test_df, train_df, validation_df = self._run_custom_split(input_df)
+        else:
+            test_df, train_df, validation_df = _run_split(
+                self.task, input_df, self.split_ratios, self.target_col
```

Comment:
This is not specific to your PR but do you know why we switched the ordering from train/validation/test to test/train/validation for these internal function returns? It can cause subtle bugs and I don't see a good reason of doing it.

Reply:
Yeah! I don't remember about that as well. I can change it to train/validation/test. I agree that seems better for sure.

Reply:
SG. Do you mind updating the ordering here for all split functions? Thanks!